### DISCUSSION TITLE
Error while adding a syslog drain

### DISCUSSION THREAD

**Discussion by fpietka** [vercel/vercel#5985] [Help] [answered]
Hi,
I'm trying to add ad log drain to get my logs onto my rsyslog server along with my other applications.
According to documentation I'm doing a `POST` request on the https://api.vercel.com/v1/integrations/log-drains endpoint with the following payload:
```json
{
	"name": "rsyslog",
	"type": "syslog",
	"url": "syslog://[my syslog address]:514",
	"projectId": "[my project ID]"
}
```

but doing so I get the following error:
```json
{
  "error": {
    "code": "bad_request",
    "message": "Only apps can subscribe to log-drains"
  }
}
```

How come I can't add my own log drains? maybe I'm missing something or doing something wrong?

**Comment by Fumler** [comment]
Just ran into the same problem trying to manually add a Coralogix log drain to an app. Anyone have some insight?

**Comment by italodeandra** [comment]
I'm currently studying how to integrate my logs with Discord, by listening to the logs on a "bridge" application and redirecting it to my channel's web hook. So here's what I found in the past days and that might be useful to you.

You probably need to do that through a integration. You can create on here: https://vercel.com/dashboard/integrations/console

You can learn more about them here: https://vercel.com/docs/integrations

And for reference, something that it took me a while to find, here's an example from Vercel's team on how to implement an log drain integration: https://github.com/vercel/integrations/tree/master/log-drains

**Comment by ecklf** [comment, marked as answer]
Hey there 👋, this is now possible with https://vercel.com/changelog/configurable-log-drains.
